e2e_samples/temperature_events/Infra/infra-ado-ci-pipelinet-test.yml

```diff
@@ -0,0 +1,28 @@
+trigger:
```

Comment:
What is the difference between this one and `infra-ado-ci-pipeline.yml`? is this the renamed file?

Reply:
infra-ado-ci-pipeline-test.yml is a validation pipeline